frontend/src/main.js

```diff
@@ -7,17 +7,14 @@ Vue.config.productionTip = false;
 Vue.config.devtools = true;
 
 // Import all locales
-import locale_da from "./i18n/da.json";
```

Comment:
See comment at the other file - it looks like you're removing other languages (probably unintended). Please fix this and update the PR.